ninja/src/test/java/controllers/ApiControllerMockTest.java

```diff
@@ -0,0 +1,32 @@
+package controllers;
+
+import static org.junit.Assert.assertEquals;
+import javax.inject.Inject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import ninja.NinjaRunner;
+import ninja.Result;
+import services.UserService;
+
+@RunWith(NinjaRunner.class)
+public class ApiControllerMockTest {
```

Comment:
same question about the test class name

Reply:
Fixed to ApiControllerMockUnitTest
Ninja only has unit test (even for integration test)